backup-index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quizzzy</title>
+    <link rel="stylesheet" href="styles/index.css" />
+    <link href="src/fontawesome/css/all.css" rel="stylesheet" />
+  </head>
+  <body class="app-grid">
+    <header class="app-grid__header centered-grid">
+      <div class="header__heading-box centered-grid">
+        <h1 class="header__heading">Quizzy</h1>
+      </div>
+    </header>
+
+    <main class="app-grid__content centered-grid">
+      <section class="quizcard__card">
+        <div class="quizcard__bookmark--box">
+          <div class="bookmark__box--inside">
+            <i class="fas fa-bookmark"></i>
+          </div>
+          <div class="bookmark__box--outside">
+            <i class="far fa-bookmark"></i>
+          </div>
+        </div>
+        <h3 class="quizcard__card--headline">Question:</h3>
```

Comment:
Hierarchically a `h2`-element would be more correct. And again, the headline is not a modifier.
```suggestion
        <h2 class="quizcard__headline">Question:</h2>
```